frontend/src/screens/Game Tutorial/GameTutorialScreens.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { ReactElement } from 'react';
+import { StyleSheet } from 'react-native';
+import GameTutorialScreen from './GameTutorialScreen';
+import { LIGHTPURPLE, TURQUOISE } from '../../constants/colors';
+import Swiper from 'react-native-swiper';
+
+// Images to display on pairing and selecting tutorial pages
+const tutorialImages = {
+	page1: require('../../assets/tutorial-page-1.png'),
+	page2: require('../../assets/tutorial-page-2.png'),
+	page3: require('../../assets/tutorial-page-3.png'),
+    page4: require('../../assets/tutorial-page-4.png'),
+    page5Pairing: require('../../assets/pairing-tutorial-page-5.png'),
+    page5Selecting: require('../../assets/selecting-tutorial-page-5.png'),
+    page6: require('../../assets/tutorial-page-6.png'),
+};
+
+interface GameTutorialScreenProps {
+    isPairing: boolean;
+    onFinish: Function;
+}
+
```

Comment:
I think we could simplify the code by handling the difference between the tutorials in `GameTutorialScreens` instead of `GameTutorialScreen`. As I understand (and please let me know if I'm wrong), there is only one tutorial screen that is different between the pairing and selecting tutorials. 

My suggestion is that we update the `GameTutorialScreen` props as follows:

```
interface GameTutorialScreenProps {
    screenContent: string;
    secondaryScreenContent?: string;
    imagePath?: Object;
    hasNavButton?: boolean;
    navButtonTitle?: string;
    onFinish: Function;
}
```

Then, for the fifth tutorial screen  (the one that differs) on `GameTutorialScreens` we could do something along the lines of:


```
{isPairing && 
<GameTutorialScreen ...appropriate pairing text and image props />}
{!isPairing &&
<GameTutorialScreen ...appropriate selecting text and image props />}
```

We can look at this together and talk about it to make sure this is the right approach!